solutions/CSES Minimum Coins.mdx

```diff
@@ -40,6 +40,12 @@ you used, and print $-1$ if so (since it's impossible to achieve a sum of $x$).
 
 </Warning>
 
+<LanguageSection>
+
+<CPPSection>
+
+## Implementation
```

Comment:
move `## Implementation` outside `<LanguageSection>` and add `**Time Complexity**: \mathcal O(N\cdot X)` below it